Blish HUD/GameServices/Gw2ChatCodeService.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Blish_HUD
+{
+    public class Gw2ChatCodeService : GameService
```

Comment:
I think I'd like to put this in a `Blish_HUD.Common` namespace almost like a util.  Perhaps `Blish_HUD.Common.Gw2.ChatCode`?  GameServices are better suited for things that need access to an update loop.  This wouldn't be good as a service - especially since everything publicly exposed is static, anyways.

Reply:
I think we should do that in a different PR, though.  The `Common` namespace will also be good if we bring things over for reading dat files in a structured way.

Reply:
Ok I move Gw2ChatCodeService to `Common.Gw2.ChatCode`
I will do this with the commit for the change regarding Gw2WndHandle in Keyboard.cs

Reply:
Thanks!